src/content/ui/animations/staggered-animations.md

```diff
@@ -115,26 +70,52 @@ You might notice the following characteristics:
 
 ![Diagram showing the interval specified for each motion](/assets/images/docs/ui/animations/StaggeredAnimationIntervals.png)
 
+### Watch the video
+
+To experience this example in action, watch the following video.
+
+<iframe width="560" height="315" src="{{site.yt.embed}}/0fFvnZemmh8" title="Watch this example of a staggered animation in Flutter" {{site.yt.set-short}}></iframe>
+
+This video renders the following animation of a single widget.
+It begins as a bordered blue square with slightly rounded corners.
+The square runs through changes in the following order:
+
+1. Fades in
+1. Widens
+1. Becomes taller while moving upwards
+1. Transforms into a bordered circle
+1. Changes color to orange
+
+After running forward, the animation runs in reverse.
+
+## Build a basic staggered animation
+
+This guide shows how to build a staggered animation in Flutter
+using the [basic_staggered_animation][] example.
+This example shows a series of sequential and overlapping animations
+of a single widget. Tapping the screen begins an animation
+that changes opacity, size, shape, color, and padding.
+
 To set up the animation:
 
-* Create an `AnimationController` that manages all of the
-  `Animations`.
-* Create a `Tween` for each property being animated.
-  * The `Tween` defines a range of values.
-  * The `Tween`'s `animate` method requires the
-    `parent` controller, and produces an `Animation`
-    for that property.
-* Specify the interval on the `Animation`'s `curve` property.
+1. Create an `AnimationController` that manages all of the
+   `Animations`.
+1. Create a `Tween` for each property being animated.
+   * The `Tween` defines a range of values.
+   * The `Tween`'s `animate` method requires the
+     `parent` controller, and produces an `Animation`
+     for that property.
+1. Specify the interval on the `Animation`'s `curve` property.
 
 When the controlling animation's value changes,
 the new animation's value changes, triggering the UI to update.
 
+### Tween a widget's width
+
 The following code creates a tween for the `width` property.
-It builds a [`CurvedAnimation`][],
-specifying an eased curve. See [`Curves`][] for
-other available pre-defined animation curves.
+It builds a [`CurvedAnimation`][], specifying an eased curve.
 
-```dart
+```dart title="Tweening the widget's width (main.dart)"
```

Comment:
The titles on code blocks aren't really meant for descriptions but rather filenames. Is the title necessary here?

Reply:
I wondered about that.